test/transform.py

```diff
@@ -29,77 +24,59 @@
     sys.path.append(blocksWorldPath)
     from blocksWorld import *
 
-imageSize = (640, 480)
-imageMode = 'L'
-imageBackground = 'white'
 
-fileType = 'PNG'
 
-points = np.array([
-    [320,  250],
-    [320,  260],
-    [320,  270],
-    [320,  280],
-])
 
 
 class test_transform(unittest.TestCase):
 
     """
     This class tests for rotation of vertices for 180 degrees
     """
-
-    def test_rotate(self):
-        fileName = sys._getframe().f_code.co_name + '.png'
-
-        # Result image for rotate
-        result_image = Image.new(imageMode, imageSize, imageBackground)
-        result_canvas = ImageDraw.Draw(result_image)
-        draw(result_canvas, points, '+')
+    points = transform_points
+    def test_rotate_transform(self):
+        """ This function is to test rotated points in 2D"""
+        # first create an empty white image
```

Comment:
add newline before comment. It's nice to separate the lines when there are comments.